content/index.ja.md

```diff
@@ -0,0 +1,700 @@
+{% include excol.html type="start" %}
+
+#### マルチメディア関連記事 {#stories-related-to-multimedia}
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Martine、聴覚障害を持つオンライン学生](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya、視覚障害を持つ上級職員](/people-use-web/user-stories/#accountant)
+-   [Kaseem、聴覚障害と視覚障害を持つ10代の若者](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+### コンテンツはさまざまな方法で提示できる {#adaptable}
+
+コンテンツの表示をユーザーが変更できるようにするためには、以下が必要です：
+
+-   見出し、リスト、テーブル、入力フィールド、およびコンテンツ構造が適切にマークアップされている
+-   情報や指示の順序が、任意の表示から独立している
+-   ブラウザと支援技術が、表示をカスタマイズするための設定を提供している
+
+この要件を満たすことで、コンテンツが正しく音声で読み上げられたり、拡大されたり、さまざまな人々のニーズや好みに合わせて適応されたりすることが可能になります。例えば、読みやすさを向上させるために、カスタムの色組み合わせ、テキストサイズ、その他のスタイリングを使用して表示されることがあります。この要件は、ページのアウトラインや要約を自動生成するなど、他の形式の適応も容易にします。これにより、人々が概要を把握したり、特定の部分により簡単に焦点を当てたりするのに役立ちます。
+
+{% include excol.html type="start" %}
+
+#### 適応性に関するアクセシビリティ要件（技術仕様へのリンク）
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+**WCAG**
+
+-   [ガイドライン 1.3 - 適応可能](https://www.w3.org/WAI/WCAG22/quickref/#adaptable)
+
+**UAAG**
+
+-   [ガイドライン 1.4 - テキスト構成](https://www.w3.org/TR/UAAG20/#gl-text-config)
+-   [ガイドライン 1.5 - 音量構成](https://www.w3.org/TR/UAAG20/#gl-volume-config)
+-   [ガイドライン 1.6 - 合成音声の構成](https://www.w3.org/TR/UAAG20/#gl-speech-config)
+-   [ガイドライン 1.7 - ユーザースタイルシートの構成](https://www.w3.org/TR/UAAG20/#gl-style-sheets-config)
+-   [ガイドライン 1.9 - 代替ビュー](https://www.w3.org/TR/UAAG20/#gl-alternative-views)
+-   [ガイドライン 1.10 - 要素情報](https://www.w3.org/TR/UAAG20/#gl-info-link)
+
+**ATAG**
+
+-   [原則 A.1: オーサリングツールのユーザーインターフェイスは適用可能なアクセシビリティガイドラインに従う](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [ガイドライン A.2.2: (オーサリングツールのユーザーインターフェイス用) 編集ビューの表示がプログラムで決定できることを保証する](https://www.w3.org/TR/ATAG20/#gl_a22)
+-   [ガイドライン A.3.7: (オーサリングツールのユーザーインターフェイス用) プレビューが市場に出回っているユーザーエージェントと同等以上にアクセシブルであることを保証する](https://www.w3.org/TR/ATAG20/#gl_a37)
+-   [パート B. アクセシブルなコンテンツの制作をサポートする](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### 順応性に関連する記事
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee、色覚異常を持つオンラインショッパー](/people-use-web/user-stories/#shopper)
+-   [Alex、繰り返しストレス障害を持つ記者](/people-use-web/user-stories/#reporter)
+-   [Ilya、視覚障害を持つ上級職員](/people-use-web/user-stories/#accountant)
+-   [Preety、注意欠陥・多動性障害およびディスレクシアを持つ中学生](/people-use-web/user-stories/#classroomstudent)
+-   [Yun、低視力、手の震え、および軽度の短期記憶障害を持つ退職者](/people-use-web/user-stories/#retiree)
+-   [Luis、ダウン症のスーパーマーケットアシスタント](/people-use-web/user-stories/#supermarketassistant)
+-   [Kaseem、聴覚障害と視覚障害を持つ10代の若者](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+### コンテンツが見やすく、聞き取りやすくなる {#distinguishable}
+
+識別可能なコンテンツは見やすく、聞き取りやすい。そのようなコンテンツには以下のようなものがある：
+
+-   色が情報を伝えたりコンテンツを識別したりする唯一の方法として使用されていない
+-   デフォルトの前景色と背景色の組み合わせが十分なコントラストを提供している
+-   ユーザーがテキストを400%まで拡大したりテキストの間隔を変更したりしても、情報が失われない
+-   テキストが小さなウィンドウ（"ビューポート"）内で再配置され、ユーザーがテキストを大きくしたときも同様である
```

Comment:
Original version:
> Text reflows in small windows (“viewports”) and when users make the text larger

@takuan517 It seems _("viewports")_ is missing in the translation. Since this term is widely used in WCAG, I think important to keep it in the translation.

Reply:
@remibetin 
fixed
https://github.com/w3c/wai-accessibility-principles/pull/50/commits/a8d19d265459b3cef23719e85b7659ac488f771d